examples/dt/neutron/CRD_files/neutron_az_ovn.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  neutron:
+    template:
+      customServiceConfig: |
+        [DEFAULT]
+        dns_domain = openstackgate.local
+        vlan_transparent = true
+        network_scheduler_driver = neutron.scheduler.dhcp_agent_scheduler.AZAwareWeightScheduler
+        dhcp_load_type = networks
+        default_availability_zones = zone-1,zone-2
+        igmp_snooping_enable = true
+        global_physnet_mtu = 9000
+
+        [ovs]
+        igmp_snooping_enable = true
+    
+        [ovn]
+        ovn_emit_need_to_frag = true
+  
+        [ml2]
+        path_mtu = 9000
+
+  nova:
+    template:
+      schedulerServiceTemplate:
+        customServiceConfig: |
+          [filter_scheduler]
+          enabled_filters = RetryFilter, AggregateInstanceExtraSpecsFilter, AvailabilityZoneFilter,ComputeFilter,ComputeCapabilitiesFilter,ImagePropertiesFilter,DifferentHostFilter,SameHostFilter,AllHostsFilter,ServerGroupAffinityFilter,ServerGroupAntiAffinityFilter
```

Comment:
IIRC, RetryFilter is not available starting from osp17, https://bugzilla.redhat.com/show_bug.cgi?id=2028171.

Reply:
Got it!, will remove in the next commit! thanks!